### ISSUE TITLE
"invalid filename" error in Ubuntu 20.04

### ISSUE THREAD

**Issue by TK-21st** [cupy/cupy#4635]
* Conditions
```
OS                           : Linux-5.4.0-65-generic-x86_64-with-debian-bullseye-sid
CuPy Version                 : 8.3.0
NumPy Version                : 1.17.4
SciPy Version                : 1.3.3
Cython Build Version         : 0.29.21
CUDA Root                    : /usr/local/cuda-11
CUDA Build Version           : 0
CUDA Driver Version          : 11020
CUDA Runtime Version         : 11020
cuBLAS Version               : 11301
cuFFT Version                : 10400
cuRAND Version               : 10203
cuSOLVER Version             : (11, 0, 2)
cuSPARSE Version             : 11301
NVRTC Version                : (11, 2)
Thrust Version               : 101000
CUB Build Version            : 101000
cuDNN Build Version          : 7104
cuDNN Version                : 7104
NCCL Build Version           : 2305
NCCL Runtime Version         : 2804
cuTENSOR Version             : None
Device 0 Name                : UNAVAILABLE
Device 0 Compute Capability  : 61
```

* Code to reproduce
```python
>>> import cupy
>>> cupy.arange(10)
```

* Error messages, stack traces, or logs

Note that this does not break the functionality of CuPy but not sure what is causing the issue.

```bash
>>> cupy.arange(10)
invalid filename
!0 = distinct !DICompileUnit(language: DW_LANG_C_plus_plus, file: !1, producer: "lgenfe: EDG 6.0", isOptimized: true, runtimeVersion: 0, emissionKind: NoDebug, enums: !2)
!1 = !DIFile(filename: "", directory: "/home/tingkai")
invalid filename
!0 = distinct !DICompileUnit(language: DW_LANG_C_plus_plus, file: !1, producer: "lgenfe: EDG 6.0", isOptimized: true, runtimeVersion: 0, emissionKind: NoDebug, enums: !2)
!1 = !DIFile(filename: "", directory: "/home/tingkai")
array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
```

**Comment by leofang** [comment]
This is an NVRTC issue that only happens on CUDA 11.2. This is fixed in #4538, which is included in the upcoming CuPy v8.5.0.

**Comment by TK-21st** [comment]
Thank you for the prompt reply! 

Closing as answered.
